scripts/gdb.py

```diff
@@ -43,6 +44,20 @@ def find_vma_base(addr):
         return start-offset
   return None
 
+def get_text_offset(file_name):
+  #command=['readelf', '-SW', file_name]
+  command="readelf -SW {}".format(file_name)
+  proc = Popen(command, stdout=PIPE, stderr=PIPE, shell=True)
+  sections, err = proc.communicate()
+  if proc.returncode != 0:
+      raise Exception("ELF Read Error")
+  for line in sections.decode('utf-8').rstrip().split('\n'):
+      if ".text" in line:
+          text_info = line.split()
+          break
+  offset=text_info[5]
```

Comment:
Add comment:
```
0 1    2    3    4       5
  [Nr] Name Type Address Off
```

Reply:
Actually there is one potential problem (number of digits could change column position):
```
0  1  2                 3                         4           5
[  1] .text             PROGBITS         0000000000000240  00000240
```
I guess I should strip the part before section name to make it simpler? Let me know your thoughts.

Reply:
I see. Then, use a regular expression